Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.py

```diff
@@ -0,0 +1,896 @@
+    """
+    params = build_submission_params(args)
+    params.update({'url-reputation': argToBoolean(args.get('url_reputation', False))})
+    return polling_submission_command(args, client, params, exe_metrics)
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:  # pragma: no cover
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    api_key = demisto.get(demisto.params(), 'credentials.password')
+    base_url = demisto.params().get('url')
+    verify_certificate = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    reliability = demisto.params().get('Reliability', DBotScoreReliability.C)
+    create_relationships = demisto.params().get('create_relationships', False)
+    command = demisto.command()
+    args = demisto.args()
+    exe_metrics = ExecutionMetrics()
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        client = Client(apikey=api_key, base_url=base_url, accept_tac=True, verify_ssl=verify_certificate, proxy=proxy,
+                        reliability=reliability, create_relationships=create_relationships)
+
+        if command == 'test-module':
+            return_results(test_module(client))
+        elif command == 'joe-is-online':
+            return_results(is_online_command(client))
+        elif command == 'joe-list-analysis':
+            return_results((list_analysis_command(client, args)))
+        elif command == 'joe-download-report':
+            demisto.results(download_report_command(client, args))
+        elif command == 'joe-download-sample':
+            demisto.results(download_sample_command(client, args))
+        elif command == 'joe-search':
+            return_results(search_command(client, args))
+        elif command == 'file':
+            return_results(file_command(client, args))
+        elif command == 'url':
+            return_results(url_command(client, args))
+        elif command == 'joe-list–lia-countries':
+            return_results(list_lia_countries_command(client))
+        elif command == 'joe-list-lang-locales':
+            return_results(lis_lang_locales_command(client))
+        elif command == 'joe-get-account-quota':
+            return_results(get_account_quota_command(client))
+        elif command == 'joe-submission-info':
+            return_results(submission_info_command(client, args, exe_metrics))
+        elif command == 'joe-submission-sample':
+            return_results(submission_sample_command(client, args, exe_metrics))
+        elif command == 'joe-submission-url':
+            return_results(submission_url_command(client, args, exe_metrics))
+        else:
+            raise NotImplementedError(f'{command} command is not implemented.')
+
+    except Exception as e:
+        update_metrics(e, exe_metrics)
+        return_results(CommandResults(execution_metrics=exe_metrics.get_metric_list()))
```

Comment:
where is `get_metric_list` defined? I don't see it

Reply:
I updated CommonServerPython.